app/assets/javascripts/miq_explorer.js

```diff
@@ -149,6 +149,14 @@ ManageIQ.explorer.focus = function(data) {
   }
 };
 
+ManageIQ.explorer.removeSand = function(data) {
+  var mainContent = $('#main-content');
+  if (mainContent) {
```

Comment:
I believe this will always be true? as $("#..") will always return an object.
easy to check in a test ;)

Reply:
Agreed, this should be `if (mainContent.length)`

Reply:
(Or no conditions, since `removeClass` won't fail on empty collection.)